### PR TITLE
Adding canonical tags to all pages

### PR DESCRIPTION
**Pull request by markus-jf**
The canonical tag is added in the head of each page, for seo reasons.

### CODE REVIEW

**Comment by markus-jf** [comment]
Added the Star conan on github button to social icons on top of the page,
Adapted mobile menu (show hamburger menu earlier - 992px screen width)

File: index.html [hunk 1]
```diff
@@ -55,12 +55,17 @@
...
-                        <img src="img/github-icon.png" width="22" height="21" alt="GitHub" />
-                    </a>
                     <a href="https://cpplang.now.sh/" target="_blank">
                         <img src="img/slack-icon.png" width="22" height="21" alt="Slack" />
                     </a>
+                    <a href="https://github.com/conan-io" target="_blank">
+                        <img src="img/github-icon.png" width="22" height="21" alt="GitHub" />
+                    </a>
+                    <div class="github-star d-inline position-relative">
+                        <a class="github-button" href="https://github.com/conan-io/conan" data-icon="octicon-star" aria-label="Star conan-io/conan on GitHub">
```
[the remark this answers is not in the document]
Comment (markus-jf) [reply index.html]: I completely agree. I didn't find a solution (yet) to directly star it. Like this it is just taking you to the github repo.

I'll revert this commit and will add the star button without the JS file.

Also, I will check about the 'Go To Github' in the middle of the page.

[the remark this answers is not in the document]
Comment (markus-jf) [reply index.html]: The star button on the top menu is now updated. Removed the external JS from all pages and added the button as svg.

**Comment by memsharded** [comment]
See comments in https://github.com/conan-io/website/pull/46

**Comment by memsharded** [comment]
Also, please better separate PR, per different thing:
- A PR for the canonical tags, etc
- A PR for the github star button
- etc

Thanks!

**Pull request merged**